build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/RunAndroidEmulatorCheckBootTimes.cs

```diff
@@ -32,9 +36,16 @@ public override bool Execute ()
 			var timeoutInMS = (int) TimeSpan.FromMinutes (15).TotalMilliseconds;
 			bool finishAsExpected = false;
 			bool processTimedOut = false;
+			string args = $"--devicename {DeviceName} --verbose";
+			if (!string.IsNullOrWhiteSpace (AndroidSdkDirectory)) {
+				args += $" --sdkpath {AndroidSdkDirectory}";
```

Comment:
For Great Sanity, this should probably be quoted somehow? Presumably:

```csharp
args += $" --sdkpath \"{AndroidSdkDirectory.TrimEnd ('\\')}\"";
```

Likewise `AvdManagerHome`, below.